pisek/jobs/status.py

```diff
@@ -29,7 +28,7 @@ def _bar(self, msg: str, part: int, full: int, color: str = "cyan") -> str:
         msg = pad(msg, MSG_LEN)
         progress_msg = f"  ({part}/{full})"
 
-        bar_len = min(terminal_width - len(msg) - len(progress_msg), MAX_BAR_LEN)
+        bar_len = TARGET_LINE_WIDTH - MSG_LEN - len(progress_msg)
```

Comment:
This has the disadvantage of having the bar length change as `part` increases, as well as nearly always having different bars have different lengths. The original logic wasn't *bad*, it was just confusingly written.
How about something like this?

```
TARGET_BAR_WIDTH = TARGET_LINE_WIDTH - MSG_LEN - 11
# ...
bar_len = min(TARGET_BAR_WIDTH, TARGET_LINE_WIDTH - len(msg) - len(progress_msg))
```

Maybe it would be a good idea to add a minimum length as well, say `10` characters.